README.md

```diff
@@ -5,24 +5,27 @@ When hosting a BAPC regional one can need:
 - For DOMjudge (https://www.domjudge.org/) 64^2 (PNG)
 - ICPC Resolver (https://tools.icpc.global/resolver/) 256^2 (PNG)
 
-| ICPC ID | Name | Status |
-|---|---|---|
+| ICPC ID | Name | Status             |
+|---|---|--------------------|
 | U-3546 | Ghent University | :heavy_check_mark: |
```

Comment:
Looks like Ghent is missing in `organisations.json` and `teams.json` (and also out-of-order) :slightly_smiling_face:

Reply:
Ghent has not participated in last 2 years